pkg/cloud_provider/file/fake.go

```diff
@@ -76,7 +76,7 @@ func NewFakeServiceForMultishare(instances []*MultishareInstance, shares []*Shar
 	return s, nil
 }
 
-func (manager *fakeServiceManager) CreateInstance(ctx context.Context, obj *ServiceInstance) (*ServiceInstance, error) {
+func (manager *fakeServiceManager) CreateInstance(ctx context.Context, obj *ServiceInstance, sourceSnapshotId string) (*ServiceInstance, error) {
```

Comment:
where is `sourceSnapshotId` being used?

Reply:
Good point. The mock has been ignoring the sourceSnapshotId, since we don't save it on the ServiceInstance object

Reply:
Fixed so the mock is receiving and saving BackupSource as part of the ServiceInstance